### DISCUSSION TITLE
New feed is useless.

### DISCUSSION THREAD

**Discussion by dohooo** [community/community#37573] [Other Feature Feedback, Questions, & Ideas]
Why force me to have the information that I don't need? I've been looking for Settings for days, trying to switch back, and I've realized that there's no way to change them.

**Comment by ossanna16** [comment]
Hi there @dohooo and welcome to our community! Thank you for the feedback 🙂

**Comment by glorialangreo** [comment]
> Why force me to have the information that I don't need? I've been looking for Settings for days, trying to switch back, and I've realized that there's no way to change them.

Hey @dohooo do you mind sharing what's the non relevant information you are seeing in the feed? We'd be happy to learn more.

**Comment by Gerrit0** [comment]
The algorithmic feed contains a bunch of stuff I just flat don't care about. If I wanted to know about someone's activity, I'd follow them.

For those stuck in the A/B testing that locks you into the algorithm: https://github.com/Gerrit0/old-github-feed

**Comment by kiyolee** [comment]
Some random guy who follows me (for which I have no control) and have the habbit of starring and following like *everything*, now my feed is essentially spamed with hundreds if not thousands of projects they starred or people they followed. There is no way to filter those or simply to switch them off. My feed is just full of noise.

**Comment by designbyadrian** [comment]
I'd rather see a list of my repos or one of my starred list when I log in.
